assets/scss/base/_helpers.scss

```diff
@@ -177,3 +177,15 @@
 	@extend %no-bullet;
 }
 
+.break-word{
```

Comment:
Can you make this a placeholder and extend it inside the selector? Then it can be used elsewhere more efficiently